### ISSUE TITLE
nan problem

### ISSUE THREAD

**Issue by qiuzhongwei-USTB** [vita-epfl/openpifpaf#112]
![image](https://user-images.githubusercontent.com/34059431/62748599-cdf33780-ba8b-11e9-958c-2fb2ebb620c0.png)

I train the model based on resnet50, I get nan problem.

This is my command line
python -m openpifpaf.train --gpus='0,1,2,3' --lr=1e-3 --momentum=0.95 --epochs=75 --lr-decay 60 70 --batch-size=32 --loader-workers 4 --basenet=resnet50block5 --head-quad=1 --headnets pif paf --square-edge=401 --regression-loss=laplace --lambdas 30 2 2 50 3 3 --freeze-base=1

I use 4 GPU, and batch size is 32.

**Comment by svenkreiss** [comment]
My guess is that your learning rate schedule is too aggressive for that batch size. Try increasing the burn-in epochs to 12.

**Comment by qiuzhongwei-USTB** [comment]
you mean that the lr rate warm up step should be enlarged with the increasing batch size? 3 epochs for batch size 812 epochs for batch size 32... if I use a large batch size by 8 or 16 GPUs, the lr is low at the whole process of training. Is that right? how about change the lr from e-3 to e-4?

**Comment by svenkreiss** [comment]
Yes. My suggestion is based on keeping the number of steps in the burn in phase the same. 4x larger batch size means 4x fewer steps per epoch. Therefore, to compensate, I suggested to dilate the schedule by 4x. 
That should get you to stable learning without Nans. 

Larger batch size implies other changes in the dynamics of the optimization that can be explored once you have stable learning. E.g. reducing or increasing learning rate.
